interface/forms/procedure_order/common.php

```diff
@@ -26,10 +26,16 @@
 use OpenEMR\Common\Csrf\CsrfUtils;
 use OpenEMR\Common\Forms\ReasonStatusCodes;
 use OpenEMR\Core\Header;
+use OpenEMR\Events\Services\LabTransmitEvent;
+use Symfony\Component\EventDispatcher\EventDispatcher;
```

Comment:
don't see why line 30 above is needed. Am I missing something?

Reply:
Because its referenced in line 36 in the php annotation this line should remain.  The dispatch will still work but php intellisense will not if the line is missing.  Another alternative is to put the FQDN on line 36, but I'd rather it be here on line 30.